### DISCUSSION TITLE
Forum

### DISCUSSION THREAD

**Discussion by X8X-XZDX** [X8X-XZDX/VALVE-3RD-TURN#6] [Announcements]
Apparently this is a thing on github

**Comment by altaltaltaltalt594-create** [comment]
Yo it's me, Emperor Mustard, I don't know if you still check this Github but if you do then please add me again my old account got banned and the new one is named emperormustarded

**Reply by X8X-XZDX** [reply]
I believe all you gotta do is just interact with the repos and you are in tada.

This will all be getting reworked and all of the projects are going to be getting rebooted ASAP.  Ping DCord
